include/rcpputils/shared_library.hpp

```diff
@@ -58,7 +58,7 @@ class SharedLibrary
   /**
    * \param[in] symbol_name name of the symbol inside the shared library
    * \return shared library symbol pointer, if the symbol doesn't exist then throws a
-   * \runtime_error exception
+   * runtime_error exception
```

Comment:
It seems like "if the symbol doesn't exist then throws a runtime_error exception" isn't necessary to say, since you repeat it on the next line.